.github/changes-filter.yaml

```diff
@@ -16,33 +16,39 @@ frontend:
   - "**/typescript_test.yml"
 docs:
   - "docs/**"
-# Test categories and their associated paths
-components:
-  - "src/frontend/src/components/**"
-  - "!src/frontend/src/components/starterProjects/**"
 
+# Test categories and their associated paths
 starter-projects:
-  - "src/frontend/src/components/starterProjects/**"
   - "src/backend/base/langflow/initial_setup/**"
-  - "src/frontend/workspace/starterProjects/**"
+  - "src/frontend/src/pages/MainPage/**"
+  - "src/frontend/src/utils/reactflowUtils.ts"
+
+components:
+  - "src/frontend/src/components/**"
+  - "src/frontend/src/modals/**"
+  - "src/frontend/src/pages/FlowPage/**"
+  - "src/frontend/src/shared/**"
+  - "src/frontend/src/hooks/**"
+  - "src/frontend/src/CustomNodes/**"
+  - "src/frontend/src/style/**"
+  - "src/frontend/src/utils/styleUtils.ts"
+  - "src/frontend/src/utils/reactflowUtils.ts"
+  - "src/frontend/src/utils/buildUtils.ts"
```

Comment:
Shouldn't buildutils and reactflow related files be in workspace?

Components should be related to the Components themselves (e.g genericNode).

Reply:
makes sense